compiler/src/main/kotlin/io/spine/protodata/backend/Documentation.kt

```diff
@@ -149,10 +154,10 @@ private constructor(private val value: List<Int>) {
          */
         fun fromMessage(descriptor: Descriptor): LocationPath {
             val numbers = mutableListOf<Int>()
-            numbers.add(FileDescriptorProto.MESSAGE_TYPE_FIELD_NUMBER)
+            numbers.add(DescriptorProtos.FileDescriptorProto.MESSAGE_TYPE_FIELD_NUMBER)
```

Comment:
There is a lot of such changes in this PR. Don't you think it's less convenient to address the types and their members by specifying a "full" (so to speak) name, rather than stripping some common parts by having them "statically" imported?